docs/index.md

```diff
@@ -2,6 +2,42 @@
 
 ### Description
 
+### Manual:
```

Comment:
Instead of writing the text to the main page, add a new page manual.md or so, and put a link on the navigation bar on the left.